### ISSUE TITLE
www folder is empty

### ISSUE THREAD

**Issue by nemchik** [linuxserver/docker-tt-rss#32]
A user on discord reported having trouble and seeing an error in their logs so I did some additional testing and here's the results.

I spun up a brand new container with the following compose:
```yaml
  ttrss:
    container_name: ttrss
    environment:
    - PGID=${PGID}
    - PUID=${PUID}
    - TZ=${TZ}
    image: linuxserver/tt-rss
    ports:
    - ${TTRSS_PORT_80}:80
    restart: unless-stopped
    volumes:
    - /etc/localtime:/etc/localtime:ro
    - ${DOCKERCONFDIR}/ttrss:/config
```
(some irrelevant parts removed like logging)

The first run had no issues. I stopped and started the container and the second run had the following error in the log:
```
[cont-init.d] 19-newdefault: executing...
mv: cannot stat '/defaults/default.incontainer': No such file or directory
[cont-init.d] 19-newdefault: exited 0.
```

I stopped and removed the container and recreated it (same config volume) and it did not have an error.

I stopped the container again and started it again (without recreating) and found the same error once again in the logs.

I checked the script in question that's causing the error and found this https://github.com/linuxserver/docker-tt-rss/blob/de8446230209af44a98f65f20eece2ef2adb027a/root/etc/cont-init.d/19-newdefault#L4-L5 which references `/config/www/tt-rss` which I do not have. I do have `/config/www` but no `tt-rss` inside it.

**Comment by nemchik** [comment]
I've pinged the user on discord to provide more information as my description is only specific to the docker logs, but their original issue that lead them to look in the docker logs actually has to do with the initial setup of the app and it not having permissions to create files/folders in `/config/www` (I think based on how they described the issue). Sounds like it wants to be `www-data` but that doesn't quite fit with how nginx is run in your containers I think.

**Comment by revers59** [comment]
I am the user on the discord

I am not very comfortable with English sorry.

I fixed the problem by recreating the containair.

Here's what I did the first time:
I create the datebase tt-rss on mariadb
I use the Power Users information in the database with my database data


    -e DB_TYPE = mysql
    -e DB_HOST = host
    -e DB_USER = user
    -e DB_NAME = name
    -e DB_PASS = password
    -e DB_PORT = 3306
    -e SELF_URL_PATH = http: // localhost /

With that I did not have access to the site.

2 / I removed all Power users info

3 / I relaunched the container and I had access to the site and I entered my information for the base in the web page dedicated to that

4 / the process went well, but I arrived on a page similar to the page mentioned on the forum tt-rss: https://discourse.tt-rss.org/t/startup-failed-permisions-777 -for-local-net / 1497

I check docker log and see : mv: cannot stat '/defaults/default.incontainer': No such file or directory

I had to erase and rebuild the container for it to work

Thanks to everyone who helped me on the discord: iXNyNe - j0nnymoe
